frontend/app/routes/$lang/_protected/alerts/index.tsx

```diff
@@ -17,17 +17,21 @@ export async function loader({ context: { session }, params, request }: LoaderFu
 
   const userInfoToken: UserinfoToken = session.get('userInfoToken');
 
-  invariant(userInfoToken.sin, 'Expected userInfoToken.sin to be defined');
+  invariant(userInfoToken.sub, 'Expected userInfoToken.sub to be defined');
```

Comment:
```suggestion
  invariant(userInfoToken?.sub, 'Expected userInfoToken.sub to be defined');
```

Reason: `userInfoToken` should never be undefined, but technically it can be.

Reply:
got this error: "Unnecessary optional chain on a non-nullish value." after i added ?

Reply:
k.. my bad.